api/net/Areas/Admin/Controllers/EventScheduleController.cs

```diff
@@ -0,0 +1,124 @@
+using System.Net;
+using System.Net.Mime;
+using Microsoft.AspNetCore.Mvc;
+using Swashbuckle.AspNetCore.Annotations;
+using TNO.API.Areas.Admin.Models.EventSchedule;
+using TNO.API.Models;
+using TNO.Core.Exceptions;
+using TNO.DAL.Services;
+using TNO.Keycloak;
+
+namespace TNO.API.Areas.Admin.Controllers;
+
+/// <summary>
+/// EventScheduleController class, provides event schedule endpoints for the api.
```

Comment:
An event schedule provides a way to automate when a report or notification is sent out.